app/views/code_of_conduct/index.html.erb

```diff
@@ -1,56 +1,61 @@
+  Use this as an opportunity to respectfully interact with a diverse group of individuals in a professional and safe environment.
 </p>
+
 <p>
-  If a participant engages in harassing behaviour, the local/global workshop team may take any action they deem appropriate, including warning the offender or expulsion from the workshop.
+  We are all here to learn and grow. This event, being a professional and safe environment, is not the appropriate venue to question individuals on their underrepresented status.
 </p>
-
-<h2>THE LESS QUICK VERSION</h2>
 <p>
-  Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention. Participants asked to stop any harassing behavior are expected to comply immediately.
+  Anyone found to be behaving in any way that makes participants uncomfortable or impedes the intended goals of this event will be in violation of this Code of Conduct and will be ejected from Global Diversity CFP Day.
 </p>
+
 <p>
-  Sponsors are also subject to the anti-harassment policy. In particular, sponsors should not use sexualized images, activities, or other material. Booth staff (including volunteers) should not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
+  Offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, unwelcome photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention are also considered inappropriate behaviour.
 </p>
 <p>
-  If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact your workshop organiser or facilitator.
+  If your behavior proves to be harmful to others you will have to be removed from all Global Diversity CFP Day events.
 </p>
 <p>
-  Workshop staff will be happy to assist those experiencing harassment to feel safe for the duration of the workshop. We value your attendance and participation.
+  If you or someone else are subjected to inappropriate behviour or have any other concerns please follow the Reporting steps below.
 </p>
+
+
+<br/>
+<h2>Reporting Steps</h2>
 <p>
-  We expect participants to follow these rules at all workshop venues.
+  Each workshop team is happy to assist attendees feel safe for the duration of the workshop. We value your participation.</p>
```

Comment:
"Each workshop team is happy to assist attendees feel safe" sounds a bit funny.

I wonder if "Each workshop team is happy to assist attendees in feeling safe" makes more sense, or a simpler "The workshop teams are here to help attendees feel safe".

Reply:
Went with "The workshop teams are here to help attendees feel safe" 

Thanks!